### PR TITLE
Fix error while daemon starting

### PR DESCRIPTION
**Pull request by johnroyer**
Error ocurs when starting daemon `s3-provider` : `Back-off restarting failed container`

Add execute permission for `docker-entrypoint.sh` can fix this error.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,123 +1,65 @@
-## Shared storage with S3 backend
-The storage is definitely the most complex and important part of an application setup, once this part is 
-completed, 80% of the tasks are completed.
+以下步驟會透過 s3-fuse 建立 docker image，並讓 k8s pod 能夠 mount AWS s3 bucket。
```
Comment (gaul) [inline README.md]: This has a bunch of unrelated changes.  Can you narrow this to only the essentials?

**Review by gaul — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (johnroyer) [reply README.md]: Oops, my bad.

Comment (johnroyer) [reply README.md]: I reverted the un-needed changes.

File: Dockerfile [hunk 2]
```diff
@@ -28,4 +28,5 @@ RUN apk --update add bash fuse libcurl libxml2 libstdc++ libgcc alpine-sdk autom
 RUN sed -i s/"#user_allow_other"/"user_allow_other"/g /etc/fuse.conf
 
 COPY docker-entrypoint.sh /
+RUN ["chmod", "+x", "/docker-entrypoint.sh"]
```
Comment (gaul) [inline Dockerfile:31]: Would it be better to use:

```
COPY --chmod=0755 docker-entrypoint.sh /
```

**Review by gaul — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (johnroyer) [reply Dockerfile:31]: I think the better way is to set file permission for `docker-entrypoint.sh` :

```
chmod +x docker-entrypoint.sh
```

----

`COPY` in Dockerfile will keep file permission. There is no further action if the file permission has set correctly.
